src/controller/model/device.ts

```diff
@@ -102,9 +101,19 @@ class Device extends Entity {
     set skipDefaultResponse(skipDefaultResponse: boolean) {this._skipDefaultResponse = skipDefaultResponse;}
     get skipTimeResponse(): boolean {return this._skipTimeResponse;}
     set skipTimeResponse(skipTimeResponse: boolean) {this._skipTimeResponse = skipTimeResponse;}
-    get defaultSendRequestWhen(): SendRequestWhen {return this._defaultSendRequestWhen;}
     set defaultSendRequestWhen(defaultSendRequestWhen: SendRequestWhen) {
```

Comment:
There are only 3 references to this in zigbee-herdsman-converters, could we refactor those and remove this entirely?

Reply:
would be fine for me, too - I just thougt we could give it a little grace period, remove the 3 occcurrences (and maybe also all occurrences of `sendWhen`) and then remove the adapters. 
Should I better remove it right away and prepare a PR for zhc?

Reply:
Let's do it right away, I will merge the PRs at the same time.

Reply:
Alright, corresponding zhc PR is here: https://github.com/Koenkk/zigbee-herdsman-converters/pull/6865